persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/PrekeyDAO.kt

```diff
@@ -13,8 +13,8 @@ internal class PrekeyDAOImpl internal constructor(
 ) : PrekeyDAO {
     override suspend fun updateOTRLastPrekeyId(newKeyId: Int) {
         metadataQueries.transaction {
-            val currentId = metadataQueries.selectValueByKey(OTR_LAST_PRE_KEY_ID).executeAsOne().toInt()
-            if (newKeyId > currentId) {
+            val currentId = metadataQueries.selectValueByKey(OTR_LAST_PRE_KEY_ID).executeAsOneOrNull()?.toInt()
+            if (currentId == null || newKeyId > currentId) {
```

Comment:
this is something i saw in logs so i added the fix here, the updateOTRLastPrekeyId is failing